### PR TITLE
Add Vencord icon

### PR DESCRIPTION
**Pull request by somerand0mcat**
vencord

**Popularity metric:**
8.7k stars at https://github.com/Vendicated/Vencord
62,084 global rank at https://www.similarweb.com/website/vencord.dev/

### Checklist

- [X] I updated the JSON data in `_data/simple-icons.json`
- [X] I optimized the icon with SVGO or SVGOMG
- [X] The SVG `viewbox` is `0 0 24 24`

### Description
there was no svg file so i had to manually convert the png file https://github.com/Vencord/Vesktop/blob/main/build/icon.png to svg in order to make the icon

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -16792,6 +16792,11 @@
             "source": "https://venmo.com/about/brand",
             "guidelines": "https://venmo.com/about/brand"
         },
+	    {
+	        "title": "Vencord",
+	        "hex": "8E71D4",
+	        "source": "https://raw.githubusercontent.com/Vencord/Vesktop/main/build/icon.png?format=webp&quality=lossless&width=662&height=662"
```
Comment (uncenter) [inline _data/simple-icons.json:16798]: This does not follow https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines.

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:1]: Please keep the indentation consistent in this file.

File: icons/vencord.svg
Comment (uncenter) [inline icons/vencord.svg:1]: This is not properly optimized as per our guidelines.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json [hunk 2]
```diff
@@ -16792,6 +16792,11 @@
             "source": "https://venmo.com/about/brand",
             "guidelines": "https://venmo.com/about/brand"
         },
+	    {
+	        "title": "Vencord",
+	        "hex": "8E71D4",
```
Comment (uncenter) [inline _data/simple-icons.json:16797]:
```suggestion
	        "hex": "D3859B",
```
`#d3859b` seems like the Vencord theme primary accent.

**Comment by somerand0mcatt** [comment]
> I also tried to convert and optimize the PNG manually, but without filling in the letters.
that looks alot more better, i'll use that svg

**Review by mondeja — APPROVED** [review]
This is an amazing work. Thank you all! 🥇

**Pull request merged**